system/modules/core/library/Contao/Files/Php.php

```diff
@@ -57,6 +57,9 @@ public function rmdir($strDirectory)
 	 */
 	public function fopen($strFile, $strMode)
 	{
+		if (is_link($strFile)) {
+			$strFile = readlink($strFile);
```

Comment:
Yes, please change it. And also ensure that the logic is the same in both the `fopen()` and the `rename()` method.